docs/_docs/collections.md

```diff
@@ -89,18 +89,59 @@ choice and written out to `<dest>/my_collection/some_subdir/some_doc.html`.
 
 ## Configuring permalinks for collections {#permalinks}
 
-You can customize the [Permalinks](../permalinks/) for your collection's documents by setting `permalink` property in the collection's configuration as follows:
+You can specify a pattern for the URLs where your Collection documents will reside with the [`permalink` property](../permalinks/):
```

Comment:
There's a modifier problem with the last "with the ..." clause. Maybe recast as "Through the [`permalink` property](../permalinks/), you can specify a pattern for the URLs for your Collection documents. The permalink also defines the structure for the files in the output."